### DISCUSSION TITLE
Support for jam.coop

### DISCUSSION THREAD

**Discussion by gribbles67** [web-scrobbler/web-scrobbler#4396] [Ideas]
Really interesting new Bandcamp replacement that aims ultimately to be an artist/community owned coop. 

https://jam.coop/

**Comment by carpet-berlin** [comment]
Yes, quite interesting. Will work on that.

**Reply by gribbles67** [reply]
Nice one - the jam.coop team will appreciate that. As do I as an artist who uses them and would like to see them grow & succeed. 

Thanks ☺️ 

.g

**Reply by carpet-berlin** [reply]
https://github.com/web-scrobbler/web-scrobbler/pull/4408

**Comment by inverse** [comment]
Looks like an interesting service - looks kinda similar to https://resonate.coop/

**Comment by chrislo** [comment]
Thank you for adding this!
